Apps/Sandcastle/gallery/Time Dynamic Point Cloud.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
```

Comment:
Trying to run this, I get "Request has failed. Status Code: 404"

Is this because the data is only in the specs?

http://cesium-dev.s3-website-us-east-1.amazonaws.com/cesium/point-cloud-stream-new/Apps/Sandcastle/index.html?src=Time%20Dynamic%20Point%20Cloud.html